build/package/scripts/build-java.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -eu
+
+OUTPUT_DIR="docker"
+WORKING_DIR="."
+ROOT_DIR=$(pwd)
+ARTIFACTS_DIR=$ROOT_DIR/.ods/artifacts
+ARTIFACT_PREFIX=
+DEBUG="false"
+GRADLE_ADDITIONAL_TASKS=
+GRADLE_OPTIONS=
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+
+    --working-dir) WORKING_DIR="$2"; shift;;
+    --working-dir=*) WORKING_DIR="${1#*=}";;
+
+    --output-dir) OUTPUT_DIR="$2"; shift;;
+    --output-dir=*) OUTPUT_DIR="${1#*=}";;
+
+    --gradle-additional-tasks) GRADLE_ADDITIONAL_TASKS="$2"; shift;;
+    --gradle-additional-tasks=*) GRADLE_ADDITIONAL_TASKS="${1#*=}";;
+
+    # Gradle project properties ref: https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties
+    # Gradle options ref: https://docs.gradle.org/current/userguide/command_line_interface.html
+    --gradle-options) GRADLE_OPTIONS="$2"; shift;;
+    --gradle-options=*) GRADLE_OPTIONS="${1#*=}";;
+
+  *) echo "Unknown parameter passed: $1"; exit 1;;
+esac; shift; done
+
+if [ "${WORKING_DIR}" != "." ]; then
+  WORKING_DIR="${ROOT_DIR}"
+  ARTIFACT_PREFIX="${WORKING_DIR/\//-}-"
+fi
+
+if [ "${DEBUG}" == "true" ]; then
+  set -x
+fi
+
+export GRADLE_USER_HOME=/home/gradle/.gradle
+echo "Exported env var 'GRADLE_USER_HOME' with value '${GRADLE_USER_HOME}'"
+echo
+cd "${WORKING_DIR}"
+echo "Working on Java/Gradle project in '${WORKING_DIR}'..."
```

Comment:
If we change the name, let's align this sentence as well.

Reply:
I'm happy to change the name if we decide to do it. In that case I would prefer to do it in a separate PR. Sounds good?